### PR TITLE
Text updates

### PR DESCRIPTION
**Pull request by owillborn**
*Issue #:* /bcgov/entity#15417

*Description of changes:*
- Minor text update in confirm completion for transfer flow
- Text changes in Confirm Completion component depending on transfer type

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by cameron-eyds — COMMENTED** [review]
Looks good, although wasn't there some Tooltip content to update as well?

**Review by doug-lovett — APPROVED** [review]
Approved.

**Comment by owillborn** [comment]
> Looks good, although wasn't there some Tooltip content to update as well?

@cameron-eyds Pls correct me if I'm wrong but I didn't see anything in the AC or design regarding a tooltip for the confirm component...

**Review by cameron-eyds — APPROVED** [review]
LGTM pending the TT and version conflicts 👍

**Review by cameron-eyds — APPROVED** [review]

File: ppr-ui/src/components/mhrTransfers/ConfirmCompletion.vue [hunk 1]
```diff
@@ -21,17 +21,61 @@
...
+                  added as an owner.</p>
                 <p class="confirm-completion-note">
                   <span>Note: </span> If the bill of sale has been signed by someone acting on behalf of the registered
                   owners, the person submitting this transfer is a lawyer or notary, and the power by which the
                   signatory was authorized was power of attorney, representation agreement, committee, receiver, or writ
                   of seizure and sale.
                 </p>
               </li>
+              <li v-else-if="transferType==='TRAND'" class="pl-3 pb-3 mb-7">
+                <p><strong>Orignal or
```
Comment (cameron-eyds) [inline ppr-ui/src/components/mhrTransfers/ConfirmCompletion.vue]: Minor typo here

**Review by cameron-eyds — APPROVED** [review]
Minor comment but otherwise g2g 👍

File: ppr-ui/src/components/mhrTransfers/ConfirmCompletion.vue [hunk 2]
```diff
@@ -21,17 +21,61 @@
...
+                  transfer to a beneficiary, you must have written consent from all beneficiaries that are not being
+                  added as an owner.</p>
                 <p class="confirm-completion-note">
                   <span>Note: </span> If the bill of sale has been signed by someone acting on behalf of the registered
                   owners, the person submitting this transfer is a lawyer or notary, and the power by which the
                   signatory was authorized was power of attorney, representation agreement, committee, receiver, or writ
                   of seizure and sale.
                 </p>
               </li>
+              <li v-else-if="transferType==='TRAND'" class="pl-3 pb-3 mb-7">
```
Comment (cameron-eyds) [inline ppr-ui/src/components/mhrTransfers/ConfirmCompletion.vue]: Oops, we also have an enum for these types 👍 
See `ApiTransferTypes`

File: ppr-ui/src/components/mhrTransfers/ConfirmCompletion.vue [hunk 3]
```diff
@@ -21,17 +21,61 @@
...
+                    If the death certificate was issued outside of Canada or the US, the transfer will have to be sent
+                    to the Manufactured Home Registry
+                  </v-tooltip>
+                  , and the name on the death certificate matches the name displayed above exactly.
+                  </p>
+              </li>
               <li class="pl-3 pb-3 mb-7">
+                <p><strong>Transfer or Change Ownership form</strong> has been recieved and retained</p>
+              </li>
+              <li v-if="transferType==='TRANS'" class="pl-3 pb-3 mb-7">
```
Comment (cameron-eyds) [inline ppr-ui/src/components/mhrTransfers/ConfirmCompletion.vue]: same as above

**Review by cameron-eyds — APPROVED** [review]

**Pull request merged**